src/frontend/src/lib/components/transactions/TransactionsDateGroup.svelte

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+	import { slide } from 'svelte/transition';
+	import { SLIDE_DURATION } from '$lib/constants/transition.constants';
+	import type { AllTransactionsUi } from '$lib/types/transaction';
+
+	export let date: string;
```

Comment:
- `date: string;` maybe a more verbose naming just because of "date"?

- Not for this PR instead of formatting the date to string in the utilitiy shouldn't actually receive a `Date` object and take care of the formatting rendering for separation of concerns reason?

Reply:
ah, for example `dateAsString` ? or something else?

RE: formatting in the component, I think you are right!, I will do it in another PR

Reply:
Given that it will become a date in another PR, I guess we can keep `date` then? or `groupDate`?

It's just that reading `date: string` earlier today felt like reading "apple: orange" 😁

Reply:
ahahahahahah yes, agreed!